docs/wallet/xportal.md

```diff
@@ -0,0 +1,92 @@
+---
+id: xportal
+title: xPortal
+---
+
+[comment]: # (mx-abstract)
+
```

Comment:
might start with a description of xPortal, a link to the website, and only after that begin with the guardians docs, because this is the only xPortal section on the website

Reply:
added description